automem/search/runtime_keywords.py

```
@@ -0,0 +1,48 @@
+        from automem.utils.text import SEARCH_STOPWORDS as _am_search_stopwords
+        from automem.utils.text import _extract_keywords as _am_extract_keywords
+
+        search_stopwords = _am_search_stopwords
+        entity_stopwords = _am_entity_stopwords
+        entity_blocklist = _am_entity_blocklist
+        return (
+            search_stopwords,
+            entity_stopwords,
+            entity_blocklist,
+            _am_extract_keywords,
+        )
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)